test/integration/scenarios/network/peer_disconnect.js

```diff
@@ -168,7 +146,10 @@ module.exports = params => {
 									return done(err);
 								}
 
-								if (numOfConnections <= expectedOutgoingConnections + 18) {
+								if (
+									numOfConnections <=
```

Comment:
Same here, we're reusing `networkParameters.EXPECTED_OUTOGING_CONNECTIONS + 18` and `networkParameters.EXPECTED_OUTOGING_CONNECTIONS - 20` few times - move to globals maybe.

Reply:
Same as above

Reply:
It can be added as a separate issue as the next following improvements -- explain hardcoded variables.